Packs/Jira/Integrations/JiraV3/JiraV3_description.md

```diff
@@ -5,6 +5,11 @@ Please configure only one of the following fields:
 
 For both instances, it is advised to use the `https://oproxy.demisto.ninja/authcode` **Callback URL**. The oproxy url is a client side only web page which provides an easy interface to copy the obtained auth code from the authorization response to the integration configuration in the authorization flow steps. Optionally: if you don't want to use the oproxy url, you may use a localhost url on a port which is not used locally on your machine. For example: <http://localhost:9004>. You will then need to copy the code from the url address bar in the response.
 
+**Notes**:
```

Comment:
It might be worth mentioning that it works for Jira Cloud without any version limitation.